src/classes/BGE_DataImportBatchEntry_CTRL.cls

```diff
@@ -143,9 +188,19 @@ public with sharing class BGE_DataImportBatchEntry_CTRL {
                 'donationAmountField' => Schema.SObjectType.DataImport__c.fields.Donation_Amount__c.getName(),
                 'expectedCountField' => Schema.SObjectType.DataImportBatch__c.fields.Expected_Count_of_Gifts__c.getName(),
                 'expectedTotalField' => Schema.SObjectType.DataImportBatch__c.fields.Expected_Total_Batch_Amount__c.getName(),
-                'requireTotalMatch' => Schema.SObjectType.DataImportBatch__c.fields.RequireTotalMatch__c.getName(),
                 'noDataImportsFound' => String.format(Label.stgNoObjectsFound, new List<String>{Schema.SObjectType.DataImport__c.getName()}),
-                'processBatchButton' => 'Process Batch'
+                'opportunityImportedLookupField' => Schema.SObjectType.DataImport__c.fields.DonationImported__c.getName(),
```

Comment:
Why are some of these getName() and others getLabel()?

Reply:
Some are used as labels/headers, so we need their localized entity label. Some are used as field references to accommodate namespacing, so we need the api name.